### PR TITLE
Added some missing intrinsic variables to the manual.

### PR DESCRIPTION
**Pull request by tnowotny**

### CODE REVIEW

File: doxygen/15_UserGuide.dox [hunk 1]
```diff
@@ -167,7 +167,19 @@ GeNN has no explicitly hard-coded synapse and neuron variables. Users are free t
 - \c DT : Time step (typically in ms) for simulation;
 Neuron integration can be done in multiple sub-steps inside the neuron model for numerical stability (see Traub-Miles and Izhikevich neuron model variations in \ref sectNeuronModels).
 
-- \c inSyn: This is an intermediary synapse variable which contains the summed input into a postsynaptic neuron (originating from the \$(addToInSyn, X) or \$(addToInSynDelay, X, Y) functions of the weight update model used by incoming synapses) . 
+\note DT exceptionally does not require bracketing with $(.)
+
+- \c t : The current time as a floating point value
```
Comment (neworderofjamie) [inline doxygen/15_UserGuide.dox:172]: in ms

File: doxygen/15_UserGuide.dox [hunk 2]
```diff
@@ -167,7 +167,19 @@ GeNN has no explicitly hard-coded synapse and neuron variables. Users are free t
...
+
+- \c id : The index of the neuron/ synapse in the population
```
Comment (neworderofjamie) [inline doxygen/15_UserGuide.dox:174]: ``id `` is only valid in the neuron context so in neurons, current sources, postsynaptic models; and pre and postsynaptic weight update models. ``id_syn`` gives some sort of index of the current synapse.

File: doxygen/15_UserGuide.dox [hunk 3]
```diff
@@ -167,7 +167,19 @@ GeNN has no explicitly hard-coded synapse and neuron variables. Users are free t
...
+
+- \c id_pre : Used in a synapse context. The index of the pre-synaptic neuron in its population.
+
+- \c id_post : Used in a synapse context. The index of the post-synaptic neuron in its population.
+
+- \c inSyn : This is an intermediary synapse variable which contains the summed input into a postsynaptic neuron (originating from the \$(addToInSyn, X) or \$(addToInSynDelay, X, Y) functions of the weight update model used by incoming synapses) .
```
Comment (neworderofjamie) [inline doxygen/15_UserGuide.dox:180]: This is only valid within postsynaptic models I think

File: doxygen/15_UserGuide.dox [hunk 4]
```diff
@@ -167,7 +167,19 @@ GeNN has no explicitly hard-coded synapse and neuron variables. Users are free t
...
+
+- \c id : The index of the neuron/ synapse in the population
+
+- \c id_pre : Used in a synapse context. The index of the pre-synaptic neuron in its population.
+
+- \c id_post : Used in a synapse context. The index of the post-synaptic neuron in its population.
+
+- \c inSyn : This is an intermediary synapse variable which contains the summed input into a postsynaptic neuron (originating from the \$(addToInSyn, X) or \$(addToInSynDelay, X, Y) functions of the weight update model used by incoming synapses) . 
+
+- \c revInSyn : Like inSyn but for "inverse" input onto the pre-synaptic neuron. Used, e.g., for error backpropagation. Is targeted through the \$(addToPre, C) function.
```
Comment (neworderofjamie) [inline doxygen/15_UserGuide.dox:182]: I don't think ``revInSyn`` is ever exposed to the user.

Comment (tnowotny) [reply doxygen/15_UserGuide.dox:182]: Happy to leave this out.

**Pull request merged**